packages/constellation/src/components/Compositions/Slider/Slider.tsx

```diff
@@ -0,0 +1,25 @@
+import * as SliderPrimitive from '@radix-ui/react-slider'
```

Comment:
Would it make this more tree-shakable if we just imported `{ Root, Track, Range, Thumb }` instead of everything from SliderPrimitive?

Reply:
Yep I think so. Will make that change now. We'll also be revisiting this component when we tackle controls. This was just our chose POC component for getting set up.